argilla-server/.env.dev

```
@@ -0,0 +1,2 @@
+ALEMBIC_CONFIG=src/argilla_server/alembic.ini
+ARGILLA_DATABASE_URL=sqlite+aiosqlite:///${HOME}/.argilla/argilla.db?check_same_thread=False
```

Disable telemetry for dev envs
```suggestion
ARGILLA_DATABASE_URL=sqlite+aiosqlite:///${HOME}/.argilla/argilla.db?check_same_thread=False
ARGILLA_ENABLE_TELEMETRY=0
```